src/app/storefront/IndividualItem.tsx

```diff
@@ -45,11 +51,20 @@ export default function IndividualItem(props: {
             style={{ width: '250px', height: '250px' }}
           />
         </ItemButtons>
-        <HeartContainer onClick={() => clickFunction()}>
-          <HeartIcon isClicked={IsFavorite} />
+
+        <HeartContainer
+          onClick={() => clickFunction()}
+          onMouseEnter={() => setHovering(true)}
+          onMouseLeave={() => setHovering(false)}
+        >
+          <HeartIcon isHovering={hovering} isClicked={IsFavorite} />
         </HeartContainer>
+        <Hover isHovering={hovering} isClicked={IsFavorite}>
+          {IsFavorite ? "Remove from favorites" : "Add to favorites"}
```

Comment:
Make sure that you are using the text components so that the text are uniform to the style from figma!